packages/pilot/src/components/DetailsHead/style.css

```
@@ -19,8 +19,8 @@
 
   & > span {
     color: var(--color-light-steel-100);
-    font-size: 12px;
-    text-transform: uppercase;
+    font-size: 14px;
+    text-transform: capitalize;
```

Podemos trocar este capitalize por um `first-letter` resolve o problema de capitalizar mais de uma palavra 😄.
```suggestion
    &:first-letter {
      text-transform: capitalize;
    }
```

---

Resolvido!